x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/actions.ts

```diff
@@ -49,7 +49,7 @@ export type Action =
       type: ACTION.SET_INDEX_PATTERN_TITLES;
       payload: {
         indexPatternTitles: State['indexPatternTitles'];
-        indexPatternsWithNumericFields: State['indexPatternsWithNumericFields'];
+        indexPatternsMap: any;
```

Comment:
based on the prop name I assume it's a `Record` or `Map` maybe

Reply:
Updated here: b2e501c17ea72ad79e742cba55bb69bf75417f44